### PR TITLE
Add domain name info to body class

### PR DESCRIPTION
**Pull request by blazer82**
I propose adding the current domain name to the body class, so it may be used in stylesheets and scripts.

### CODE REVIEW

File: multiple-domain/multiple-domain.php [hunk 1]
```diff
@@ -46,6 +46,12 @@
 $domainLang = $multipleDomain->getDomainLang();
 
 
+/**
```
Comment (straube) [inline multiple-domain/multiple-domain.php]: There is a method called `initFilters` inside the MultipleDomain class. Could you move this `add_filter` call there, please?

**Review by straube — CHANGES_REQUESTED** [review]
Please just change the location of your `add_filter` call. Otherwise, the PR looks good.

**Comment by straube** [comment]
Thanks for the contribution, BTW! 👍

File: multiple-domain/multiple-domain.php (hunk 1 above)
Comment (blazer82) [reply multiple-domain/multiple-domain.php]: I haven't noticed that method, thanks for pointing it out! I moved the `add_filter` call and updated my PR.

Thanks for the quick reply, btw 😊

**Review by straube — APPROVED** [review]
All good! Will merge the PR

**Comment by straube** [comment]
@blazer82 Do you have an account on WP.org? If so, let me know your username and I'll be happy to add you as a contributor.

**Comment by blazer82** [comment]
@straube I don't have an account there but thanks! And thanks for accepting my pull request.

**Pull request merged**